Testscripts/Linux/perf_apache.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+
+#######################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+#######################################################################
+
+#######################################################################
+#
+# perf_apache.sh
+# Description:
+#    Install apache and run apache performance tests.
+#    This script needs to be run on client VM.
+#
+# Supported Distros:
+#    Ubuntu/Suse/Centos/RedHat/Debian
+#######################################################################
+
+CONSTANTS_FILE="./constants.sh"
```

Comment:
You are not using this variable here and in another file. Please review.

Reply:
Thanks, Ju. Have fixed.